web-console/src/dialogs/status-dialog/status-dialog.tsx

```diff
@@ -99,7 +99,7 @@ export class StatusDialog extends React.PureComponent<StatusDialogProps, StatusD
             filterable
             defaultFilterMethod={(filter, row) => {
               const id = filter.pivotId || filter.id;
-              return row[id] !== undefined ? String(row[id]).includes(filter.value) : true;
+              return row[id] !== undefined ? String(row[id]).includes(filter.value) : false;
```

Comment:
Is it worth adding a unit test for this?

Reply:
I've added one test to check if the filter function is working properly.